Chatto/Source/ChatController/BaseChatViewController+Changes.swift

```diff
@@ -128,7 +130,10 @@ extension BaseChatViewController {
                              changes: CollectionChanges,
                              updateType: UpdateType,
                              completion: @escaping () -> Void) {
-
+        guard let collectionView = self.collectionView else {
```

Comment:
Please check this method carefully, cause it too complicated to get what blocks should I call on exit.

P.S. It is a good candidate for refactoring

Reply:
There are many places that requires refactoring :) Someday we will take care about them